app/pods/cases/case/subcases/subcase/overview/route.js

```diff
@@ -17,22 +27,35 @@ export default class CasesCaseSubcasesSubcaseOverviewRoute extends Route {
           id: this.case.id,
         },
       },
+      page: {
+        number: params.page,
+        size: params.size,
+      },
       sort: '-created',
     });
-    this.siblingSubcases = allSubcases.filter(
-      (subcase) => subcase.id !== model.id
-    );
-    this.mandatees = (await model.mandatees).sortBy('priority');
-    this.submitter = await model.requestedBy;
+    const siblingSubcases = allSubcases.filter((sibling) => sibling.id !== subcase.id);
+    this.siblingSubcasesCount = allSubcases.meta.count;
```

Comment:
When you query the backend, Ember keeps the `meta` property from the response in the resulting object. Normally we use this property, specifically `meta.count` to display the total amount of results.

When calling `.filter()` here, the result we get is a plain JS array, so we lose the count information and we have to store that "next" to the actual array. Ideally at some point it just becomes possible to query the backend and filter out the result we don't want so we don't need to do any filtering in the frontend.

Reply:
Another option would be to show something special to indicate which of the subcases is the one being looked at instead of just hiding it.